doc/install.rst

```diff
@@ -332,12 +332,10 @@ Bats
 Bats ("Bash Automated Testing System") is a test framework for tests written
 as Bash shell scripts.
 
-`Upstream Bats <https://github.com/sstephenson/bats>`_ is unmaintained, but
-widely available. Both version 0.4.0, which tends to be in distributions, and
-upstream master branch (commit 0360811) should work. There is a maintained
-fork called `Bats-core <https://github.com/bats-core/bats-core>`_, but the
-test suite currently does not pass with it; see `issue #582
-<https://github.com/hpc/charliecloud/issues/582>`_. Patches welcome!
+Previously, the test suite relied on an unmaintained, `legacy version <https://github.com/sstephenson/bats>`_
+of Bats (v0.4.0). As of release 0.32, we will be dropping support for this
+older version in favor of `Bats-core <https://github.com/bats-core/bats-core>`_,
+specifically v1.2.1 or greater. 
```

Comment:
1. Again you don't need to explain the history, just the current state.
2. Avoid mentioning version numbers; they're automatically inserted from `configure`, which is the source of truth on that.